### ISSUE TITLE
Can we revert the properties filetype to a csv file again?

### ISSUE THREAD

**Issue by dsmurrell** [cambDI/camb_old#51]
I think this is easier for new users to R. Since you modified it to be a tab delimited, can you change it back to being a .csv again please.

**Comment by dsmurrell** [comment]
Also, there seem to be two output files in the C++ code... I kinda go a bit crazy when I see code left like this... 
![screenshot 2014-05-28 12 12 47](https://cloud.githubusercontent.com/assets/4035854/3103597/1efafbac-e659-11e3-8f7a-08c303aa3346.png)

Can you remove the unneeded one?

**Comment by isidroc** [comment]
you already fixed this, right?

**Comment by dsmurrell** [comment]
nope, I don't want to revert something that you changed. I think you
changed it to be tab delimited.
